apps/datahub/src/app/dataset/dataset-visualisation/dataset-visualisation.component.ts

```diff
@@ -18,23 +18,25 @@ import { DatasetDistribution } from 'geonetwork-ui/libs/common/domain/src/lib/mo
 export class DatasetVisualisationComponent {
   @Input() displayMap: boolean
   @Input() displayData: boolean
-  selectedView = 'map'
```

Comment:
I think I would keep this and remove the changes in line 26 inside the constructor as it could also be the 'chart' option, right?